frontmacs-editing.el

```diff
@@ -27,6 +28,13 @@
 ;; undo visual tree
 (global-undo-tree-mode 1)
 
+;; setup flycheck
+(setq flycheck-indication-mode 'right-fringe)
```

Comment:
We should be more descriptive with our descriptions. The key here is to ask "could someone follow along here?" If there's a relevant link, it's good to include it.